packages/linting/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@department-of-veterans-affairs/eslint-config-mobile",
-  "version": "0.12.0",
+  "version": "0.12.1-alpha.0",
```

Comment:
Think this'll need to be manually bumped up? Alert will probably be components v0.14.0, right?